python.d/mysql.chart.py

```diff
@@ -224,6 +247,7 @@ def slave_running(value):
     'threads': {
         'options': [None, 'mysql Threads', 'threads', 'threads', 'mysql.threads', 'line'],
         'lines': [
+            ['var_max_connections', 'max', 'absolute'],
```

Comment:
Do we need this on chart?
I see its can be[ hidden](https://github.com/firehol/netdata/wiki/External-Plugins#dimension) (`giving the keyword hidden will make this dimension hidden, it will take part in the calculations but will not be presented in the chart`)

Reply:
I was on the fence since it's technically not a thread (just the limit) bit I think it helps give the chart more context, and you can quickly glance at it and gauge how close you are to reaching your connection limit.

Reply:
Sorry for the double reply, I'm on a road trip with the family so I can look for myself, but is there a way to add this as a gauge? Set the peak for the gauge as the max_connection_limit and the needle would be the number of connection threads.

Reply:
>  it helps give the chart more context, and you can quickly glance at it and gauge how close you are to reaching your connection limit.

sounds good 👍